### ISSUE TITLE
[ISSUE] Is GEOS-Chem correctly using MERRA-2 precipitation fields?

### ISSUE THREAD

**Issue by cdholmes** [geoschem/geos-chem#1454]
### What institution are you from?
Florida State University

### Description of the problem
There are inconsistencies in the precipitation rate implied by different MERRA-2 variables used by GEOS-Chem. The surface precipitation rate provided by PRECTOT is 30-40% greater than implied by the 3-D variables (DQR* and REEVAP*) used in GEOS-Chem for large-scale wet scavenging. 

MERRA-2 provides precipitation information in a few different forms:
* PREC(TOT,LSC,CON,ANV) - precipitation rate at the surface (TOT=total, LSC=large scale, CON=convective, ANV=anvil)
* PF(I,L)(CU,LSAN) - precipitation falling through the bottom of each model grid cell (I=ice, L=liquid), (CU=convective, LSAN= large scale and anvil)
* DQR(CU,LSAN) - precipitation formation in each grid cell (CU=convective, LSAN=large scale and anvil)
* REEVAP(CU,LSAN) - precipitation evaporation in each grid cell (CU=convective, LSAN=large scale and anvil)
The wet scavenging routines in GEOS-Chem use the PF*, DQR* and REEVAP* variables and I think the PREC* variables are just for informational purposes (not used). 

Some of these are consistent. For example, it checks out that PRECTOT = PFICU+PFILSAN+PFLCU+PFLLSAN for level 1 of the PF* variables. The DQR* and REEVAP* variables are not consistent with the others, however, as shown in these figures.

Surface precipitation from DQR* and REEVAP* (top) and PRECTOT (bottom). Scales are the same.
![image](https://user-images.githubusercontent.com/2147719/197283082-2893f29d-821d-4bb8-8a43-a58190441353.png)
![image](https://user-images.githubusercontent.com/2147719/197283171-72126368-dbff-4127-bcac-77578c29be75.png)

Fractional difference (relative to PRECTOT)
![image](https://user-images.githubusercontent.com/2147719/197283424-52b11bb0-9455-41f2-810d-58767419e112.png)

### Questions and implications
1. Is there an explanation for PRECTOT precipitation being 30-40% greater than implied by DQR* and REEVAP*? I.e. is this expected behavior?
2. Wet scavenging in GEOS-Chem may be missing 30-40% of global precipitation, when using MERRA-2, meaning that the large-scale wet scavenging should perhaps be faster than currently simulated. 
3. GEOS-Chem could ignore the DQR* and REEVAP* variables and calculate wet deposition from PF* variables, which would be consistent with PRECTOT.

### Description of troubleshooting performed
I ran a 1-month simulation with StateMet diagnostics on (Jan 2020, but shorter would show the same result). The following command will sum the DQR* and REEVAP* variables and provide surface precip in units of mm/day (same as PRECTOT):
```ncap2 -s 'precipCN=((Met_DQRCU-Met_REEVAPCN)*Met_AD).total($lev)/AREA*86400; precipLS=((Met_DQRLSAN-Met_REEVAPLS)*Met_AD).total($lev)/AREA*86400;precip=precipCN+precipLS' GEOSChem.StateMet.20200101_0000z.nc4 tmp.nc```

**Comment by bzhangnia** [comment]
Hi Chris,

Thanks for bringing this up!  We'd like to share some relevant experience in this issue:

1. Is there an explanation for PRECTOT precipitation being 30-40% greater than implied by DQR* and REEVAP*? I.e. is this expected behavior?

A couple of years ago, we compared DQR/REEVAP with the differences of PFs at the top/bottom of some random grid cells in the archived MERRA-2 files used to drive GEOS-Chem. At that time, we expected they should be similar but did find some inconsistences between the two throughout the column.  The differences were not as large as 30-40% in the few comparisons we made, so we did not follow up on this. We didn't look into PRECTOT as it is not used in the wet dep calculation. Is this expected behavior? As far as we know, yes, but we don't know exactly what processes other than DQR and REVAP are involved in determining PFs and PRECTOT. 

2. Wet scavenging in GEOS-Chem may be missing 30-40% of global precipitation, when using MERRA-2, meaning that the large-scale wet scavenging should perhaps be faster than currently simulated.

We tested this in GOCART / GEOS-5 GCM where we've used both ways (DQR/REEVAP vs. precip/reevap calculated from PFs) in our Pb-210 simulations. The wet deposition is faster in the simulation when precip/reevap is calculated from PFs. Global mean tropospheric Pb-210 lifetime reduced from 7.6 to 7.2 days. GEOS-Chem will very likely behave the same way.  

3. GEOS-Chem could ignore the DQR* and REEVAP* variables and calculate wet deposition from PF* variables, which would be consistent with PRECTOT.

Before figuring out the reason for this inconsistency, we agree that it is better to use PFs to calculate precipitation formation/reevap in wet deposition.  

Bo & Hongyu
